plugins/json-logger/README.md

```diff
@@ -0,0 +1,27 @@
+# @amplication/plugin-<name>
+
+[![NPM Downloads](https://img.shields.io/npm/dt/@amplication/plugin-auth-basic)](https://www.npmjs.com/package/@amplication/plugin-auth-basic)
+
+A short description of the plugin and its actions.
+
+## Purpose
+
+What is the purpose of this plugin and what exactly does it do.
+
+## Configuration
+
+If a configuration is required, add it here.
```

Comment:
Add information in the form a small `description` - this will be used as a description of the plugin in Amplication plugin overview aswell as on the website - the `purpose` for the plugin and lastly in the `configuration` part provider some additional context on the settings provided through the `.amplicationrc.json` file.